template/.readthedocs.yaml

```diff
@@ -4,4 +4,4 @@ python:
     - method: pip
       path: .
       extra_requirements:
-        - docs
+        - dev
```

Comment:
This looks like a regression, probably you were using a branch from before #387?

Reply:
Maybe check to make sure any other PRs (if any) since your fork were not reverted either.

Reply:
Yes, you are right, PR #387 happened 2 weeks ago, after I have cloned the repo.

Reply:
This is fixed now.